ui/kotlinx-coroutines-javafx/src/JavaFxConvert.kt

```diff
@@ -27,11 +26,11 @@ import kotlinx.coroutines.flow.*
 @ExperimentalCoroutinesApi // Since 1.3.x
 public fun <T> ObservableValue<T>.asFlow(): Flow<T> = callbackFlow<T> {
     val listener = ChangeListener<T> { _, _, newValue ->
-        try {
-            offer(newValue)
-        } catch (e: CancellationException) {
-            // In case the event fires after the channel is closed
-        }
+        /*
+         * Do not propagate an exception to the caller, it
+         * already should've been handle by the downstream
+         */
+        trySend(newValue)
```

Comment:
Could you please take a look @dkhalanskyjb?

Reply:
Makes sense.